### PR TITLE
fix: test edit and create page for items

### PR DESCRIPTION
**Pull request by Glitterrosie**
Added tests for **creation** and **editing** of items, also used some pre-generated tests. They are still 4 pending tests which we suggest to use later in development. We also added **validation** for the item.

### CODE REVIEW

File: spec/features/items/edit_spec.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+
+describe "new item page", type: :feature do
```
Comment (gwauge) [inline spec/features/items/edit_spec.rb:3]: should be called "edit item page"

File: spec/requests/items_spec.rb [hunk 2]
```diff
@@ -89,7 +89,7 @@
   describe "PATCH /update" do
     context "with valid parameters" do
       let(:new_attributes) {
-        skip("Add a hash of attributes valid for your model")
+        { name: "Item 2", description: "Item 2 description" }
       }
 
       it "updates the requested item" do
```
Comment (gwauge) [inline spec/requests/items_spec.rb:95]: add assertion after patching (e.g. name was changed correctly)

**Review by gwauge — CHANGES_REQUESTED** [review]

File: spec/requests/items_spec.rb (hunk 2 above)
Comment (simon-weissmueller) [reply spec/requests/items_spec.rb:95]: Schwör ma

**Review by gwauge — APPROVED** [review]
LGTM!

**Pull request merged**